src/app/work-item/work-item-list/work-item-list-entry.component.scss

```diff
@@ -0,0 +1,22 @@
+@import "../../../assets/stylesheets/custom";
+
+.type span{
```

Comment:
I did not remove that intentionally. But looking at the code, I would think someone or something has optimized the SASS code. This:

```
.type {
  span{
    ...
  }
}
```

is the semantically same as the version in the PR, the PR version is just optimized. The history says it was never optimized. I don't know where this came from, maybe the old, original CSS or some build has actively changed the file. 
Anyway, the optimized version is imho the better one anyway.